docs/pywbemclicommands.rst

```diff
@@ -1111,13 +1105,19 @@ Connection select command
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
 The ``connection select`` command selects a connection from the connections
-file and marks that connection as the default connection. The default
-connection will be used if it is defined and pywbemcli is started with no
-:ref:`--server general option` or :ref:`--mock-server general option` general options. A connection may be
-selected to be default either by using the NAMEargument or if no argument is
-provided by selecting from a list presented on the console. The following
-example shows changing connection from within the interactive mode of
-pywbemcli:
+file sets that connection as the :ref:`current connection` and optionally the
+:ref:`default connection` (``--default option``).
```

Comment:
Change "(``--default option``)" to "by using the ``--default`` option" (ideally with a link to the option section)

Reply:
DONE